shared/lib/transactions-controller-utils.js

```diff
@@ -36,11 +36,11 @@ export function toPrecisionWithoutTrailingZeros(n, precision) {
 
 /**
  * @param {number|string|BigNumber} value
- * @param {number} decimals
+ * @param {number=} decimals
```

Comment:
Is the `=` a typo?

Reply:
It indicates an optional param via jsdoc 
https://jsdoc.app/tags-param